backend/src/helpers/string/create-token/create-token.helper.ts

```diff
@@ -0,0 +1,5 @@
+import jwt from 'jsonwebtoken';
```

Comment:
Let's move it to the `backend/src/helpers/token/create-token/create-token.helper.ts`